spyder_terminal/terminalplugin.py

```diff
@@ -47,17 +51,17 @@
 LOCATION = osp.realpath(osp.join(os.getcwd(),
                                  osp.dirname(__file__)))
 WINDOWS = os.name == 'nt'
-
-# class TerminalConfigPage(PluginConfigPage):
-#     """Terminal plugin preferences."""
-#     pass
+_ = get_translation('spyder_terminal')
 
 
 class TerminalPlugin(SpyderPluginWidget):
     """Terminal plugin."""
 
     URL_ISSUES = ' https://github.com/spyder-ide/spyder-terminal/issues'
     CONF_SECTION = 'terminal'
+    CONFIGWIDGET_CLASS = TerminalConfigPage
+    CONF_DEFAULTS = CONF_DEFAULTS
+    # CONF_VERSION = CONF_VERSION
```

Comment:
Why is this line commented? I think you should uncomment it.

Reply:
The SpyderPluginWidget doesn't support a CONF_VERSION constant

Reply:
Ok, we should address that for Spyder 5.